plugin-bazel/src/main/kotlin/org/jetbrains/bazel/languages/bazelrc/elements/BazelrcElementTypes.kt

```diff
@@ -0,0 +1,16 @@
+package org.jetbrains.bazel.languages.bazelrc.elements
+
+import com.intellij.lang.ASTNode
+import com.intellij.psi.PsiElement
+import org.jetbrains.bazel.languages.bazelrc.psi.BazelrcLine
+
+object BazelrcElementTypes {
+  val LINE = BazelrcElementType("LINE")
+
+  fun createElement(node: ASTNode): PsiElement =
+    when (node.elementType) {
+      LINE -> BazelrcLine(node)
```

Comment:
The RHS is the same in both cases

Reply:
Yea. I wanted to default to something known for now. I'm still working on a a full PsiTree structure (I want to probably make the flag parsing a little better so I would need more nodes). 

This is temporary.